content/docs/setup/kubernetes/install/cni/index.md

```diff
@@ -39,7 +47,7 @@ replaces the functionality provided by the `istio-init` container.
     1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
         settings required.
 
-1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/helm-install/) to include the installation of the Istio CNI plugin. For example:
+1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/) to include the installation of the Istio CNI plugin. For example:
```

Comment:
a warning here about what goes wrong when this isn't done may be warranted.  (The automatic sidecar injector fails to work properly)

Reply:
Beyond the scope of the PR.